lib/improver/cli/wet_bulb_temperature.py

```diff
@@ -71,13 +71,47 @@ def main(argv=None):
                         ' iterations, the solution is accepted.')
 
     args = parser.parse_args(args=argv)
+    # Load Cubes
     temperature = load_cube(args.temperature)
     relative_humidity = load_cube(args.relative_humidity)
     pressure = load_cube(args.pressure)
+    # Process Cube
+    result = process(temperature, relative_humidity, pressure,
+                     args.convergence_condition)
+    # Save Cube
+    save_netcdf(result, args.output_filepath)
+
+
+def process(temperature, relative_humidity, pressure,
+            convergence_condition=0.05):
+    """Module to generate wet-bulb temperatures.
+
+    Call the calculate_wet_bulb_temperature function to calculate wet-bulb
+    temperatures. This process function splits input cubes over vertical levels
+    to mitigate memory issues when trying to operate on multi-level data.
 
-    result = (WetBulbTemperature(precision=args.convergence_condition).
+    Args:
+        temperature (iris.cube.Cube):
+            Cube of air temperatures (K).
+        relative_humidity (iris.cube.Cube):
+            Cube of relative humidities (%, converted to fractional)
+        pressure (iris.cube.Cube):
+            Cube of air pressure (Pa).
```

Comment:
These units are not required, there is conversion within the plugin, though the conversion is to the units you have shown (K, fraction, and Pa).